docs/en/latest/plugins/grpc-transcode.md

```diff
@@ -104,31 +129,26 @@ print(resp.status_code)
 print(resp.text)
 ```
 
-Create proto:
+This script will take in a `.pb` file and the `id` to create, encodes the content of the proto to `base64`, and calls the Admin API with this encoded content.
+
+To run the script:
 
 ```bash
 chmod +x ./upload_pb.pb
-./upload_pb.py proto.pb 1
-# 200
-# {"node":{"value":{"create_time":1643879753,"update_time":1643883085,"content":"CmgKEnByb3RvL2ltcG9ydC5wcm90bxIDcGtnIhoKBFVzZXISEgoEbmFtZRgBIAEoCVIEbmFtZSIeCghSZXNwb25zZRISCgRib2R5GAEgASgJUgRib2R5QglaBy4vcHJvdG9iBnByb3RvMwq9AQoPcHJvdG8vc3JjLnByb3RvEgpoZWxsb3dvcmxkGhJwcm90by9pbXBvcnQucHJvdG8iPAoHUmVxdWVzdBIdCgR1c2VyGAEgASgLMgkucGtnLlVzZXJSBHVzZXISEgoEYm9keRgCIAEoCVIEYm9keTI5CgpUZXN0SW1wb3J0EisKA1J1bhITLmhlbGxvd29ybGQuUmVxdWVzdBoNLnBrZy5SZXNwb25zZSIAQglaBy4vcHJvdG9iBnByb3RvMw=="},"key":"\/apisix\/proto\/1"},"action":"set"}
 ```
 
-## Attribute List
-
-| Name      | Type                                                                           | Requirement | Default | Valid | Description                      |
-| --------- | ------------------------------------------------------------------------------ | ----------- | ------- | ----- | -------------------------------- |
-| proto_id  | string/integer                                                                 | required    |         |       | `.proto` content id.             |
-| service   | string                                                                         | required    |         |       | the grpc service name.           |
-| method    | string                                                                         | required    |         |       | the method name of grpc service. |
-| deadline  | number                                                                         | optional    | 0       |       | deadline for grpc, ms            |
-| pb_option | array[string([pb_option_def](#use-pb_option-option-of-grpc-transcode-plugin))] | optional    |         |       | protobuf options                 |
+```
+./upload_pb.py proto.pb 1
+```
 
-## How To Enable
+Response:
 
-Here's an example, to enable the grpc-transcode plugin to specified route:
+```
+# 200
+# {"node":{"value":{"create_time":1643879753,"update_time":1643883085,"content":"CmgKEnByb3RvL2ltcG9ydC5wcm90bxIDcGtnIhoKBFVzZXISEgoEbmFtZRgBIAEoCVIEbmFtZSIeCghSZXNwb25zZRISCgRib2R5GAEgASgJUgRib2R5QglaBy4vcHJvdG9iBnByb3RvMwq9AQoPcHJvdG8vc3JjLnByb3RvEgpoZWxsb3dvcmxkGhJwcm90by9pbXBvcnQucHJvdG8iPAoHUmVxdWVzdBIdCgR1c2VyGAEgASgLMgkucGtnLlVzZXJSBHVzZXISEgoEYm9keRgCIAEoCVIEYm9keTI5CgpUZXN0SW1wb3J0EisKA1J1bhITLmhlbGxvd29ybGQuUmVxdWVzdBoNLnBrZy5SZXNwb25zZSIAQglaBy4vcHJvdG9iBnByb3RvMw=="},"key":"\/apisix\/proto\/1"},"action":"set"}
```

Comment:
Do we need to format this?

Reply:
The encoded string is long anyways and it would look almost the same with horizontal scrolling even if we format.